### PR TITLE
updated layout as per new mobile design, still need to get social sharing popup working

### PR DESCRIPTION
**Pull request by dorono**
OK, layout is done as per the attached screenshot, however...the behavior for the "SHARE" button dropdown at 1-column view is still not done. I think this PR is a move forward in terms of getting the responsive layout down, so if it looks OK I say merge and I can come back to the SHARE button dropdown tomorrow or in a few days. 
screenshot 2017-03-24 14 40 28

### CODE REVIEW

File: src/components/EventDetails/EventDetails.js [hunk 1]
```diff
@@ -108,16 +120,28 @@ class EventDetails extends Component {
...
-              <a href={shareUrl} className={styles.facebookBtn}>
-                <FaFacebookSquare size={25} />
-                <span>Share on Facebook</span>
-              </a>
-              <a href={shareUrl} className={styles.twitterBtn}>
-                <FaTwitter size={25} />
-                <span>Share on Twitter</span>
-              </a>
+              <button
+                onClick={this.showHideSocialBtns}>
```
Comment (pfarnach) [inline src/components/EventDetails/EventDetails.js]: I haven't pulled the branch to check, but doesn't this need a `.bind(this)` with it?

**Comment by pfarnach** [comment]
Looks great! 

Two quick things:
1.) See my inline comment
2.) Did you run this through the linter?  I think there's a couple things it will bark at.  If we want to change those rules we definitely can but it'd be nice to avoid having linter warnings build up.  Currently it's mostly based off of [AirBNB's react/a11y](https://github.com/airbnb/javascript/tree/master/packages/eslint-config-airbnb) style guides right now

File: src/components/EventDetails/EventDetails.js (hunk 1 above)
Comment (dorono) [inline src/components/EventDetails/EventDetails.js]: I did that on line 48. Or was there somewhere else I should have that?

Comment (pfarnach) [inline src/components/EventDetails/EventDetails.js]: Oh sorry I missed that up there, looks good!

**Comment by dorono** [comment]
Thanks, @pfarnach - Good call about the linting, will keep it running from now on!

**Pull request merged**